gui/screens/onboarding.py

```diff
@@ -211,7 +211,15 @@ def input_seed_phrase(self, seed_phrase: typing.List[str]):
 
         for index, word in enumerate(seed_phrase, start=1):
             self._seed_phrase_input_text_edit.real_name['objectName'] = f'statusSeedPhraseInputField{index}'
-            self._seed_phrase_input_text_edit.text = word
+            if type_of_selection == 'auto':
+                self._seed_phrase_input_text_edit.text = word
+            elif type_of_selection == 'enter':
+                word_letters = []
+                for letter in word:
+                    word_letters.append(letter)
+                word_to_put = ''.join(word_letters[:-1])
```

Comment:
Valentina, the string is a list of chars you don't need to convert it into the list
just use `word[:-1]`

Reply:
God, feel that I'm stupid :( Fixed